config/profiles_ccdl.config

```
@@ -0,0 +1,80 @@
+// CCDL-specific settings to run samples for ScPCA with internal paths
+
+// AWS batch profiles
+batch {
+  includeConfig 'profile_awsbatch.config'
+}
+batch_manual {
+  includeConfig 'profile_awsbatch_manual.config'
+}
+
+// CCDL path profiles
+ccdl {
+  params {
+    // input files
+    run_metafile = 's3://ccdl-scpca-data/sample_info/scpca-library-metadata.tsv'
+    sample_metafile = 's3://ccdl-scpca-data/sample_info/scpca-sample-metadata.tsv'
+    cellhash_pool_file = 's3://ccdl-scpca-data/sample_info/scpca-multiplex-pools.tsv'
+    project_celltype_metafile = 's3://ccdl-scpca-data/sample_info/scpca-project-celltype-metadata.tsv'
+
+    // output base directory
+    outdir = "s3://nextflow-ccdl-results/scpca/processed"
+
+    // paths to output folders, redefined to use `outdir` above
+    checkpoints_dir = "${params.outdir}/checkpoints"
+    results_dir = "${params.outdir}/results"
+
+    // a set of run_ids for testing. used only by the main workflow
+    // one single-cell with bulk, one CITE, one spatial, one multiplexed
+    run_ids = "SCPCS000001,SCPCS000050,SCPCS000203,SCPCL000537"
+
+    // include a sample project for merging. used only by the merged workflow
+    merge_run_ids = "SCPCP000016"
```

```suggestion
    // include all runs in a project when creating merged object
    merge_run_ids = "All"
```
This parameter can only be used to specify run, library, or sample IDs to include in a merged project. To actually run the merge workflow you need a `project`, which we don't set by default here. The purpose of this parameter is to specify if you only want to include a certain set of libraries from a project in the merged object.

We could set SCPCP000016 as the default project to use if we wanted to have one pre-set for testing? It just will also get used when testing the main workflow, so I don't think it's entirely necessary.